### PR TITLE
El 1483 grafana dashboards

### PR DESCRIPTION
**Pull request by starswan**
## What does this pull request do?

Pods and ingress dashboards for LAALAA

## Any other changes that would benefit highlighting?

Add /metrics endpoint to LAALAA

## Checklist

- [ ] Provided JIRA ticket number in the title, e.g. "LGA-152: Sample title"

### CODE REVIEW

**Review by HettieS — COMMENTED** [review]
Some small questions and one naming tweak!

File: kubernetes_deploy/staging/pods-dashboard.yml [hunk 1]
```diff
@@ -0,0 +1,642 @@
...
+              "intervalFactor": 2,
+              "legendFormat": "{{pod}}",
+              "refId": "E"
+            }
+          ],
+          "thresholds": [],
+          "timeFrom": null,
+          "timeRegions": [],
+          "timeShift": null,
+          "title": "Requests per Minute",
```
Comment (HettieS) [inline kubernetes_deploy/staging/pods-dashboard.yml:238]: I'm not sure why this shows multiple lines for the same pods - are you able to explain to me? Thanks!

Comment (starswan) [reply kubernetes_deploy/staging/pods-dashboard.yml:238]: Thanks - just come back to it, and I wasn't passing (http) status to a metric-per-status

File: kubernetes_deploy/staging/ingress-dashboard.yml [hunk 2]
```diff
@@ -0,0 +1,756 @@
...
+              "format": "time_series",
+              "interval": "",
+              "intervalFactor": 2,
+              "legendFormat": "",
+              "range": true,
+              "refId": "C",
+              "step": 30
+            }
+          ],
+          "title": "Pod counts by status",
```
[the remark this answers is not in the document]
Comment (starswan) [reply kubernetes_deploy/staging/ingress-dashboard.yml:527]: It was a copy paste from CFE - I think I've fixed it now

**Review by HettieS — APPROVED** [review]
Approved although for some reason your re-name of the ingress dashboard hasn't come through. Other than fixing that up, looking good 👍

**Pull request merged**